charts/kubewarden-crds/templates/policyreports.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Values.installPolicyReportCRDs }}
```

Comment:
I guess this is fine, isn't it?

Reply:
I do not think to. We should allow the user to skip the CRDs installation. The user may have the CRDs installed already.

Reply:
We can workaround that by apply a suggestion readding the line and fix the update script afterwards.